### ISSUE TITLE
Cannot attach aria-hidden="true" attribute to fa icon field

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/distill#112]
When using Font Awesome icon for RSS in `_site.yml`, you may use the following syntax:

``` yaml
# _site.yml
name: "reproducible-finance-with-r"
title: "Reproducible Finance with R"
description: |
  Exploring reproducible finance with the R statistical 
  computing environment.
base_url: https://beta.rstudioconnect.com/content/3776/
navbar:
  right:
    - icon: fa fa-rss
      href: index.xml
output: distill::distill_article
```

However, the rss fa icon is not accessible for a screen reader user without `aria-hidden="true"` attribute.

According to [this official guideline](https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility), the accessible html code for fa icon would be as follows:

``` html
    - icon: <i class="fa fa-rss" aria-hidden="true"></i>
```

My question is about how I could implement the above code for my _site.yml for rss icon.

I am one of the screen reader users, so this feature is very critical to me for accessibility.

**Comment by jjallaire** [comment]
Thanks for reporting this. It needs to be addressed within the package (and now should be). You can also include a "text" attribute and that will be used as the aria-label for the enclosing`<a>` tag (as per the spec you linked to)
